services/service.go

```diff
@@ -318,24 +331,38 @@ func (s *Service) checkNodeAuthorization(nodeID *models.NodeID, ot creds.Operato
 	return nil
 }
 
-func (s *Service) validateSignedRequest(msg *[]byte, sig *[]byte, ot pb.OperatorType) (*common.Address, error) {
+func (s *Service) validateSignedRequest(msg *[]byte, sig *[]byte, address *common.Address, ot pb.OperatorType) (*common.Address, error) {
```

Comment:
hmm, no sense in using pointers for neither `msg` nor `address` nor `sig` here. Return a normal common.Address as well.

Reply:
Done, pushed